scripts/01_prep_inputs/0100-extract-inputs.R

```diff
@@ -819,7 +821,11 @@ hab_fish_collect_prep <- habitat_confirmations %>%
   purrr::pluck("step_2_fish_coll_data") %>%
   dplyr::filter(!is.na(site_number)) %>%
   # select(-gazetted_name:-site_number) %>%
-  dplyr::distinct(reference_number, method_number, haul_number_pass_number, .keep_all = T) ##added method #
+  dplyr::distinct(reference_number, method_number, haul_number_pass_number, .keep_all = T) %>%
+  # distinct(reference_number, .keep_all = T) %>%
+  arrange(reference_number) %>%
+  mutate(across(c(date_in,date_out), janitor::excel_numeric_to_date)) %>%
+  mutate(across(c(time_in,time_out), chron::times))
```

Comment:
do we use `chron` elsewhere?  can't remember. would think we could use lubridate and not need that package but wouldnt doubt if I put it ther ein the first place.

Reply:
see #48